pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright Contributors to the Open Cluster Management project
+
+###############################################################################
+# Licensed Materials - Property of IBM
+# (C) Copyright IBM Corporation 2020 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+###############################################################################
+# Copyright (c) 2020 Red Hat, Inc
+
+apiVersion: v1
+description: A helm chart for deploying the cluster lifecycle
+name: cluster-lifecycle
+appVersion: "2.5"
+version: 2.5.0
```

Comment:
Is there some rules about the version for the addon chart?  should we consider to keep the same as the acm version? we always forget to update it in each release.

Reply:
That version field isn't going to have any effect on the resources deployed. It isn't something we need worry to keep up to date.